modules/gdscript/gdscript_editor.cpp

```diff
@@ -88,7 +89,17 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 					   "# Declare member variables here. Examples:\n"
 					   "# var a%INT_TYPE% = 2\n"
 					   "# var b%STRING_TYPE% = \"text\"\n"
-					   "\n"
+					   "\n";
+	
+	List<StringName> _classes_inherit_from_node;
+	ClassDB::get_inheriters_from_class("Node", &_classes_inherit_from_node);
```

Comment:
If I'm not mistaken, the `ClassDB.is_parent_class(class: String, inherits: String)` method is better suited for these purposes. Accordingly, the loop is not needed.

```gdscript
# GDScript example
print(ClassDB.is_parent_class("Sprite", "Node")) # True
print(ClassDB.is_parent_class("Object", "Node")) # False
```

Reply:
I didn't know that existed and this is my first PR on Godot ever. Sorry about that.

Reply:
Fixed!